modules/mesh_com/Dockerfile

```diff
@@ -47,8 +47,10 @@ RUN apt-get update -y && apt-get install -y --no-install-recommends \
     rfkill \
     iproute2 \
     python3-pip \
+    vim \
```

Comment:
This is only for debugging

Reply:
Yes, Wanted to keep it till we verify the working and then remove in the final step